test/e2e/page-objects/pages/hardware-wallet/connect-hardware-wallet-page.ts

```diff
@@ -46,9 +46,13 @@ class ConnectHardwareWalletPage {
     await this.driver.clickElement(this.continueButton);
   }
 
-  async openConnectLedgerPage(): Promise<void> {
-    console.log(`Open connect trezor page`);
+  async clickConnectLedgerButton(): Promise<void> {
+    console.log(`Click connect ledger button`);
```

Comment:
Very nit: Similar comment, could be capitalized.
```suggestion
    console.log(`Click connect Ledger button`);
```